src/NetscriptFunctions.ts

```diff
@@ -1564,7 +1565,12 @@ setRemovedFunctions(ns, {
 });
 
 export function NetscriptFunctions(ws: WorkerScript): NSFull {
-  return NSProxy(ws, ns, [], { args: ws.args.slice(), pid: ws.pid, enums });
+  return NSProxy(ws, ns, [], {
+    args: ws.args.slice(),
+    pid: ws.pid,
+    stdin: new PortHandle((ws.pid * -1) as PortNumber),
```

Comment:
Adding things as properties of NS (as opposed to functions) means they have to be created for every script, slowing down script startup for everyone (regardless of if they are used). Also, they're harder to find in the docs. So, without a pressing reason I think this ought to just be a function.

Reply:
I have it as ns.getStdin() now.